### DISCUSSION TITLE
Carmelite Rite (1938)

### DISCUSSION THREAD

**Discussion by paxdei42** [DivinumOfficium/divinum-officium#3400] [Ideas]
Laudetur Jesus Christus! 
Recently I've received the Divine Office according to the Carmelite Rite of 1938. It does not feature (yet) on divinumofficium.com, and I would like to contribute. I have had no success contacting the mail address I found on the website, which is why I post here. I do not have any experience with github, so I don't know if I could contribute personally, but I am willing to learn. Otherwise I can provide the texts. Since it's from 1938, it follows the Divino Afflatu psalter and most of its rubrics (while retaining some of its own rubrics).

**Comment by mbab** [comment]
Github is easy just need some practice.
Including new _version_  will require also some code changes apart adding texts. Do you have some programming knowledge?
Don't worry if not.
First please reveal basic differences _Carmelite rite_ to _Roman_ or may by to _Monastic_.
(eg. hours Psalms schemes, feasts ranks rules)
